bedrock/settings/base.py

```diff
@@ -885,6 +885,10 @@ def _is_bedrock_custom_app(app_name):
 BASKET_TIMEOUT = config("BASKET_TIMEOUT", parser=int, default="10")
 BASKET_SUBSCRIBE_URL = f"{BASKET_URL}/news/subscribe/"
 
+# Foundation newsletters are handled through Campaign Monitor
+FOUNDATION_URL = config("FOUNDATION_URL", default="https://kmq73rfvbh.execute-api.us-east-2.amazonaws.com")
```

Comment:
This goes to a test email list at the moment. When are we expecting the prod one? And are we sure we want to merge code that isn't production ready? (Maybe we're sure and we want to do testing. Just want to double check).